crochetcraft/svelte.config.js

```diff
@@ -12,6 +12,9 @@ const config = {
         // If your environment is not supported or you settled on a specific environment, switch out the adapter.
         // See https://kit.svelte.dev/docs/adapters for more information about adapters.
         adapter: adapter(),
+        alias: {
+            '$components': 'src/components/*'
+        }
```

Comment:
Hmm, is this a standard thing? In my own SvelteKit projects, I just put all the components into `$lib` along with the other TS files. Components are still relatively easy to distinguish from TypeScript files, since they have a different file extension.

Reply:
no, I just felt like it